pxtlib/docsrender.ts

```diff
@@ -436,11 +436,11 @@ namespace pxt.docs {
                 return '<li>' + text + '</li>\n';
             }
             renderer.heading = function (text: string, level: number, raw: string) {
-                let m = /(.*)#([\w\-]+)\s*$/.exec(text)
+                let m = /(.*)[#@]([\w\-]+)\s*$/.exec(text)
```

Comment:
I don't think @fullscreen is meant to be a anchor ID. Can you explain your change?

Reply:
Just want ``@fullscreen`` to not show in doc view of tutorial page.

Currently it's like this:

```
<h2 id="introduction-fullscreen">Introduction @fullscreen</h2>
```
![image](https://user-images.githubusercontent.com/27789908/32079086-203f05b2-ba5e-11e7-89c0-195f17daf59b.png)

Is there a better solution?

Reply:
Oh, you mean the anchor ID stuff, sorry. I just noticed that the ID's of the routing tags are single work but now I get your point. I'll remove that concatenation.